incubator/group/msg.go

```diff
@@ -2,26 +2,33 @@ package group
 
 import (
 	"bytes"
+	"fmt"
 
+	"github.com/cosmos/cosmos-sdk/codec/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	"github.com/gogo/protobuf/jsonpb"
+	proto "github.com/gogo/protobuf/proto"
 	"github.com/pkg/errors"
 )
 
 const (
-	msgTypeCreateGroup           = "create_group"
-	msgTypeUpdateGroupAdmin      = "update_group_admin"
-	msgTypeUpdateGroupComment    = "update_group_comment"
-	msgTypeUpdateGroupMembers    = "update_group_members"
-	msgTypeCreateGroupAccountStd = "create_group_account"
-	msgTypeVote                  = "vote"
-	msgTypeExecProposal          = "exec_proposal"
+	msgTypeCreateGroup        = "create_group"
+	msgTypeUpdateGroupAdmin   = "update_group_admin"
+	msgTypeUpdateGroupComment = "update_group_comment"
+	msgTypeUpdateGroupMembers = "update_group_members"
+	msgTypeCreateGroupAccount = "create_group_account"
+	msgTypeVote               = "vote"
+	msgTypeExecProposal       = "exec_proposal"
 )
 
 type MsgCreateGroupAccountI interface {
-	GetBase() MsgCreateGroupAccountBase
-	GetDecisionPolicy() StdDecisionPolicy
+	GetAdmin() sdk.AccAddress
+	GetGroup() GroupID
+	GetComment() string
+
+	GetDecisionPolicy() DecisionPolicy
+	SetDecisionPolicy(DecisionPolicy) error
```

Comment:
Returning an `error` is probably unnecessary now because we're not packing into a `oneof`